### DISCUSSION TITLE
Is there a way to select multiple tasks?

### DISCUSSION THREAD

**Discussion by slerman12** [facebookresearch/hydra#2134] [General]
Some applications can support a dynamic number of tasks potentially. For example, multi-task learning, which can involve 1 or more selected tasks. Is there a way to pass this in to Hydra? Right now, I can only configure one task, as per the ones defined in my `./task` directory, via `task=`

**Comment by Jasha10** [comment]
Perhaps via a [sweep](https://hydra.cc/docs/next/tutorials/basic/running_your_app/multi-run/) over tasks?
```commandline
python my_app.py -m task=task1,task2,task3
```

**Comment by omry** [comment]
https://hydra.cc/docs/patterns/select_multiple_configs_from_config_group/
